src/com/facebook/buck/cli/PublishCommand.java

```diff
@@ -50,6 +50,8 @@
   public static final String REMOTE_REPO_SHORT_ARG = "-r";
   public static final String INCLUDE_SOURCE_LONG_ARG = "--include-source";
   public static final String INCLUDE_SOURCE_SHORT_ARG = "-s";
+  public static final String INCLUDE_JAVADOC_LONG_ARG = "--include-javadoc";
+  public static final String INCLUDE_JAVADOC_SHORT_ARG = "-jd";
```

Comment:
A short string should be a single letter :) Pydoc uses "-w", which I think is free, to mean "generate html docs". That seems appropriate.